src/kex_ntru/kex_ntru.c

```diff
@@ -53,25 +53,10 @@ static uint8_t get_entropy_from_dev_urandom(ENTROPY_CMD cmd, uint8_t *out) {
 		return 1;
 	}
 	if (cmd == GET_BYTE_OF_ENTROPY) {
-#if defined(WINDOWS)
-		// NOTE (Christian Paquin): I added this so it works on Windows. 1st, I think this
-		// should be part of OQS's rand lib, and 2nd why are the entropy bytes obtained
-		// one by one?
-		HCRYPTPROV hCryptProv;
-		int r = 1;
-		if (!CryptAcquireContext(&hCryptProv, NULL, NULL, PROV_RSA_FULL, CRYPT_VERIFYCONTEXT) ||
-		    !CryptGenRandom(hCryptProv, r, out)) {
-			r = 0;
-		}
-#else
-		int fd = open("/dev/urandom", O_RDONLY);
-		if (fd <= 0)
-			return 0;
-		int r = read(fd, out, 1);
-		close(fd);
-#endif
-		if (r != 1)
+		// TODO: why is this called to get entropy bytes one by one?
```

Comment:
I was following some example code and that's what they did.  I'm not sure why.  If it can be changed without causing errors, then we can do that.  I'll accept your PR as it is though.